force-app/main/default/pages/STG_PanelAlloBatch.page

```diff
@@ -1,5 +1,27 @@
 <apex:page controller="STG_PanelAlloBatch_CTRL" docType="html-5.0" standardStylesheets="false">
-    <apex:form id="form" styleClass="slds-m-around_x-large">
+    <apex:includeLightning/>
+    <script>
+        if ({!NOT(isRunningUserIsAdmin)}) {
+            $Lightning.use("{!namespace}" + ":RD2_EnablementApp", function () {
+                $Lightning.createComponent("{!namespace}" + ":utilIllustration",
+                    {
+                        title: "{!$Label.commonAdminPermissionErrorTitle}",
+                        message: "{!$Label.commonPermissionErrorMessage}",
+                        size: 'small',
+                        variant: 'no-access',
+                        illustrationClass: "slds-p-top_x-large slds-m-top_x-large"
+                    },
+                    "stgPanelErrIllustration",
```

Comment:
Minor comment. These div ids are specifically calling out the "stgPanelErr" vf page. Don't think it's a blocker but ideally, we'd make this a generic id or update them to be specific to each page.